src/main/resources/explanations/challenge32.adoc

```diff
@@ -0,0 +1,7 @@
+=== A Secret uploaded to an LLM (Large Language Model) application
+
+AI is here to stay. AI can help with loads of things: whether it comes to complicated problem-solving, generating possible codes, or giving advice on security.
+But what if you uploaded a password as part of sharing some code to look for a security issue. Can this secret be extracted, even when it has been protected by rules of the LLM not to share that secret with you?
```

Comment:
```suggestion
Imagine you've uploaded code to look for security issues. What if that included a password? Rules should limit the LLM model to prevent it from leaking the password. Can this secret be extracted nonetheless?
```